formats/email_alert_signup/publisher/details.json

```diff
@@ -13,17 +13,7 @@
       "minProperties": 1,
       "properties": {
         "tags": {
-          "type": "object",
-          "description": "DEPRECATED: The tags used to match subscribers lists",
```

Comment:
I left these out as I wasn't certain they applied to all uses of tags. Happy to put them back in.